File: HappyWarehouse.Domain/Dto/WarehouseItemDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HappyWarehouse.Domain.Dto
{
    public class WarehouseItemDto
    {
        public int Id { get; set; }
        public string ItemName { get; set; } = string.Empty;
        public string? SkuCode { get; set; }
        public int Qty { get; set; }
        public decimal CostPrice { get; set; }
        public decimal? MsrpPrice { get; set; }
        public int WarehouseId { get; set; }
        public string WarehouseName { get; set; } = string.Empty;
    }
}

File: HappyWarehouse.SPA/src/app/models/warehouse-item.dto.ts
export interface WarehouseItemDto {

  id: number;
  itemName: string;
  skuCode?: string | null;
  qty: number;
  costPrice: number;
  msrpPrice?: number | null;
  warehouseId: number;
  warehouseName: string;

}

export interface WarehouseItemFormModel {
  itemName: string;
  skuCode: string;
  qty: number;
  costPrice: number;
  msrpPrice: number;
}
